code/_onclick/click.dm

```diff
@@ -234,6 +234,7 @@
 		var/mob/living/carbon/human/H = user
 		H.dna.species.grab(H, src, H.martial_art)
 		H.next_click = world.time + CLICK_CD_MELEE
+	else ..()

```

Comment:
not for anything.
the only time two statements are allowed on the same line is in macros.